nixos/modules/services/wayland/cage.nix

```
@@ -36,8 +36,18 @@ in {
     type = types.path;
     default = "${pkgs.xterm}/bin/xterm";
     defaultText = literalExpression ''"''${pkgs.xterm}/bin/xterm"'';
+    example = literalExpression ''
+      pkgs.writeShellScript "cage-program" \'\'
```

[doc](https://nixos.org/manual/nix/stable/language/values.html?highlight=escape#type-string)

```suggestion
      pkgs.writeShellScript "cage-program" '''
```

---

Thanks for the link, I wasn't sure where the escape rules could be found.